dvc/state.py

```diff
@@ -7,11 +7,12 @@
 import logging
 
 from dvc.config import Config
-from dvc.utils import remove, current_timestamp, relpath
+from dvc.utils import remove, current_timestamp, relpath, chunk
 from dvc.exceptions import DvcException
 from dvc.utils.fs import get_mtime_and_size, get_inode
 from dvc.utils.compat import fspath_py35
 
+SQLITE_MAX_VARIABLES_NUMBER = 999
```

Comment:
Maybe we can retrieve it with sqlite3_limit ?

Reply:
According to [docs](https://www.sqlite.org/c3ref/limit.html) `sqlite3_limit` is for setting the limit. Also I cannot find this method in python libs.

Reply:
> Regardless of whether or not the limit was changed, the sqlite3_limit() interface returns the prior value of the limit.